src/js/components/Grid/__tests__/Grid-test.js

```diff
@@ -60,6 +60,24 @@ test('Grid areas renders', () => {
   expect(tree).toMatchSnapshot();
 });
 
+test('Grid areas renders when given an array of string arrays', () => {
```

Comment:
Thank you for adding a test! 
Will you be able to add a quick storybook example that demonstrates the new option for the `area`?   It will help new users that would like to explore different area inputs.

Reply:
Sure ! I was not sure if I should add one, I'll do that 👍

Reply:
I added a storybook example